modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -325,4 +332,117 @@ export default class NestedTabs extends Base {
 			}
 		}
 	}
+
+	getActiveClass() {
+		const settings = this.getSettings();
+
+		return settings.classes.active;
+	}
+
+	getVisibleTabTitle( tabTitleFilter ) {
+		const $tabTitle = this.elements.$tabTitles.filter( tabTitleFilter ),
+			isTabTitleDesktopVisible = null !== $tabTitle[ 0 ]?.offsetParent,
+			$visibleTabTitle = isTabTitleDesktopVisible ? $tabTitle[ 0 ] : $tabTitle[ 1 ];
+
+		return $visibleTabTitle;
```

Comment:
The `visibleTabTitle` variable is redundant.
```suggestion
			isTabTitleDesktopVisible = null !== $tabTitle[ 0 ]?.offsetParent;
			return isTabTitleDesktopVisible ? $tabTitle[ 0 ] : $tabTitle[ 1 ];
	}
```

Reply:
Updated.